src/platform/datapath_winuser.c

```diff
@@ -1672,7 +1708,7 @@ SocketCreateUdp(
             setsockopt(
                 SocketProc->Socket,
                 IPPROTO_IP,
-                IP_ECN,
+                IP_RECVTOS,
```

Comment:
I think this might be causing test failures because packets aren't getting received in some tests since adding this change.